dev/fattest.simplicity/src/componenttest/rules/repeater/JakartaEE9Action.java

```diff
@@ -178,33 +178,45 @@ public static void transformApp(Path appPath) {
         //create backup directory
         Path serverPath = appPath.getParent().getParent();
         Path backupPath = serverPath.resolve("backup");
-            try {
-                if(!Files.exists(backupPath)){
-                    Files.createDirectory(backupPath); // throws IOException
-                }
-             } catch(IOException e){
-                 Log.info(c, m, "Unable to create backup directory.");
-                 Log.error(c, m, e);
-                 throw new RuntimeException(e);
-             }
+        try {
+            if (!Files.exists(backupPath)) {
+                Files.createDirectory(backupPath); // throws IOException
+            }
+        } catch (IOException e) {
+            Log.info(c, m, "Unable to create backup directory.");
+            Log.error(c, m, e);
+            throw new RuntimeException(e);
+        }
+
+        String userDir = System.getProperty("user.dir");
+        String transformerRulesRoot = userDir.substring(0, userDir.lastIndexOf("/dev/")) + "/dev/wlp-jakartaee-transform/rules/";
```

Comment:
I don't think this will work on remote builds because generally only artifacts within the autoFVT.zip are accessible to the test at runtime. There are also a few other special cases like build.image, fattest.simplicity, ant_build, and test_build projects, but wlp-jakartaee-transform is not one of them.

To work around this, try adding some code to `fattest.simplicity/build.gradle` that copies these transform rules into `autoFVT-templates/` at build time (see the `assembleBoostrap` task as a similar example). This will result in the transform rules being included in the autoFVT.zip so you can find them under `user.dir` at bucket runtime without needing to walk up any directories into unknown territory.

Reply:
Notice the solution in fat.gradle, which copies the transformer rules files along with the transformer jar.